### DISCUSSION TITLE
PR Review Party - Feedback

### DISCUSSION THREAD

**Discussion by Meldiron** [appwrite/appwrite#2163] [General]
On 8.9.2021 we ran our first PR Review Party on our [Discord server](https://appwrite.io/discord) 🥳  In a 1-hour long session, two of our developers went live and showed you how we review your Hacktoberfest pull requests.

### We are really happy to introduce this session to our community and we would like to hear your opinion!

Please share your feedback and let's discuss these topics:

- Overall, did you like our session?
- What can we do to improve the session?
- Would you like to see more of these?

**Comment by bsantosh909** [comment]
Just loved it! It really feels great to see how the entire review process is like.

And yeah few more of these sessions would be really awesome.

**Comment by 2002Bishwajeet** [comment]
This was my First PR Review Session. Absolutely loved it the way you reviewed  🔥.

I would love to attend weekly PR review sessions.

One suggestion/request . It would be great if we see every maintainer how they review PRs. We saw Matej and Brandon how they reviewed and it was awesome, would love to see from other maintainers too

**Comment by stnguyen90** [comment]
I loved the session. I really appreciated how y'all went through the PR slowly and explained what to look for and how to go about doing a PR. There's not a lot of material/help/resources on real world tasks like doing PRs.

As useful as it is, I'm worried about doing more frequent sessions because it takes away valuable time from maintainers.

**Comment by ramdevcm** [comment]
Great session.... Got an idea on how you review PR's... 
Need more sessions like this🥰

**Comment by achintya-7** [comment]
I was not able to see the whole session but can you make a document or readme for making effecive PRs

**Comment by asmitbm** [comment]
The session was really great. I loved it. It was very insightful on how maintainers review PRs. This would help me in creating more better PRs in future. I liked how maintainers explained stuff and gave suggestions. 

Hope we'll get these sessions after hacktoberfest too. ✌🏼

**Comment by harshita214** [comment]
PR review party session was really interesting, did’nt know reviewing PR consist of soo many steps.All the reviewers did a fabulous job by reviewing PR accurately.

looking ahead for more such sessions 🤩

**Comment by AmreshSinha** [comment]
Today was my first PR Review Session. Found it interesting as I learned many things and yeah @Meldiron 's and @christyjacob4 's review speed is :zap: :sunglasses: 

Waiting for more such sessions! :+1:

**Reply by Shweta200126** [reply]
Same, I joined late but it's interesting to join and learn!
